content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -152,14 +152,29 @@ The following table shows which referential action each database supports.
 
 | Database    | Cascade | Restrict | NoAction | SetNull | SetDefault |
 | :---------- | :------ | :------- | :------- | :------ | :--------- |
-| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
+| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**⌘ | **✔️**     |
 | MySQL       | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**†    |
 | SQLite      | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
 | SQL Server  | **✔️**  | **❌**‡  | **✔️**   | **✔️**  | **✔️**     |
 | CockroachDB | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
 | MongoDB††   | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **❌**     |
 
-- † On MySQL, `SetDefault` is ignored and is silently replaced with `NoAction` instead.
+<Admonition type="warning">
+
+Although referential actions are an ANSI SQL feature, some relational databases diverged from the standard, giving rise to a few special cases.
+
+</Admonition>
+
+### Special cases for referential actions
+
+- † On MySQL (using the InnoDB storage engine), `SetDefault` is not supported, with differences depending on the database version. On MySQL 8+, `SetDefault` is considered an alias for `NoAction`.
```

Comment:
> On MySQL 8+, `SetDefault` is considered an alias for `NoAction`.

Is that really the case? Isn't it more that it is not explicitly implemented, and falls back to the default referential action (which is `NoAction`)?

Reply:
As things stand on MySQL 8+, defining a referential action as `SetDefault` or `NoAction` is de facto equivalent. So, regardless of whether that's intentional and the reasons behind it, one is an alias for the other

Reply:
An alias implies that this is an implemented state. I think it is an accidental function of `SetDefault` not (correctly) being implemented. The "considered" is also a weird way to say this in this situation.

Something like "`SetDefault` effectively acts as an alias for `NoAction`". would fit better.

Reply:
I like that wording. Nice